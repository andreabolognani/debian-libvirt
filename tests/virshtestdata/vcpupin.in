echo Invalid syntax.
vcpupin test a 0,1

echo An out-of-range vCPU number deserves a diagnostic, too.
vcpupin test 100 0,1

echo Negative number
vcpupin test -100 0,1

echo missing argument
vcpupin test --cpulist 0,1

echo An out-of-range vCPU number when get information with live flag
vcpupin test 100 --live

echo An out-of-range vCPU number when get information without flag
vcpupin test 100

echo An out-of-range vCPU number when get information with config flag
vcpupin test 100 --config

echo An out-of-range vCPU number when get information with current flag
vcpupin test 100 --current
