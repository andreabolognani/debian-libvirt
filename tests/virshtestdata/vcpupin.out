Invalid syntax.
error: Numeric value 'a' for <vcpu> option is malformed or out of range

An out-of-range vCPU number deserves a diagnostic, too.
error: invalid argument: requested vcpu '100' is not present in the domain

Negative number
error: Numeric value '-100' for <vcpu> option is malformed or out of range

missing argument
error: vcpupin: Missing vCPU number in pin mode.

An out-of-range vCPU number when get information with live flag
error: vcpu 100 is out of range of live cpu count 2

An out-of-range vCPU number when get information without flag
error: vcpu 100 is out of range of live cpu count 2

An out-of-range vCPU number when get information with config flag
error: vcpu 100 is out of range of persistent cpu count 2

An out-of-range vCPU number when get information with current flag
error: vcpu 100 is out of range of live cpu count 2


## Exit code: 1
